compiler/rustc_error_codes/src/error_codes/E0373.md

```diff
@@ -50,3 +50,25 @@ fn foo() -> Box<Fn(u32) -> u32> {
 
 Now that the closure has its own copy of the data, there's no need to worry
 about safety.
+
+This error may also be encountered while using `async` blocks:
+
+```compile_fail,E0373
+use std::sync::Arc;
+use tokio::runtime::Runtime; // 0.3.1
+
+async fn f() {
+    let room_ref = Arc::new(Vec::new());
```

Comment:
nit: style-wise, these examples usually use more "bland" variable names to put the focus on the code structure. `use()`, `x`, `y`, `foo`, `task` or `task_handle`, etc.